src/app/components/schedule-card/ScheduleCard.scss

```diff
@@ -66,7 +67,7 @@
 		margin-left: -5px;
 		border-width: 5px;
 		border-style: solid;
-		border-color: transparent transparent #f6d4d4dc transparent;
+		border-color: transparent transparent $peach transparent;
```

Comment:
maybe we can just convert this to a `border-bottom-color` rule since we're only working with the bottom border in this

Reply:
so the peach one should be border-bottom-color, just making sure bc it looks a little funky when i do that:

![image](https://github.com/WICS-UCI/venushacks/assets/108845502/2c7c3745-b47a-45ea-a8a0-5551fb1222fd)

Reply:
mmm do you mean the black square - i'm leaning towards keeping it for now then but that styling will def have to be overhauled next year so they don't run into a can of worms when styling either

Reply:
yea im not sure what to do about that either, everything else has been fixed though

Reply:
okay we can leave it then LOL. i'll add a note to look into tooltip styling in the future but otherwise we'll [animate and] merge 🥳

Reply:
would it be okay to merge now since animation would be nice but i don't have the time to do it, also merging into main right?

Reply:
yup! i'll merge it now